src/TetherComponent.jsx

```diff
@@ -94,6 +94,20 @@ class TetherComponent extends Component {
     if (this._targetComponent && this._elementComponent) {
       this._createContainer();
     }
+
+    // Verify if className props have changed
+    if (this._elementParentNode && this.props.className !== props.className) {
```

Comment:
Copied over from the other commit in (https://github.com/box/box-ui-elements/pull/1673)

This is in the `componentWillUpdate` to avoid refactoring `_update()` and closely mimics the other PR even though that one uses `componentDidUpdate`.